nifi-assembly/LICENSE

```diff
@@ -1487,3 +1487,56 @@ This project bundles 'Jython' which is available under a Python Software Foundat
     OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
     NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
     SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+This project bundles ‘Mockito’ which is available under an MIT license
```

Comment:
This should not be added to the Assembly License. Mockito is a test dep only and thus should not be included in the Licensing of the binary distribution.